infra/terraform/modules/data_access/users/tests/test_delete_user_role.py

```diff
@@ -2,21 +2,25 @@
 
 import users
 
+from tests.conftest import ROLE_NAME
```

Comment:
Should I name all these constants with the `TEST_` prefix for consistency? What do you reckon @andyhd ?